docs/sources/send-data/promtail/cloud/eks/_index.md

```diff
@@ -49,14 +52,22 @@ To ship all your pods logs we're going to set up [Promtail]({{< relref "../../..
 
 What's nice about Promtail is that it uses the same [service discovery as Prometheus][prometheus conf], you should make sure the `scrape_configs` of Promtail matches the Prometheus one. Not only this is simpler to configure, but this also means Metrics and Logs will have the same metadata (labels) attached by the Prometheus service discovery. When querying Grafana you will be able to correlate metrics and logs very quickly, you can read more about this on our [blogpost][correlate].
 
-Let's add the Loki repository and list all available charts.
+Let's add the Loki repository and list all available charts. To add the repo, the following command:
```

Comment:
```suggestion
Let's add the Loki repository and list all available charts. To add the repo, run the following command:
```

Reply:
Whoops, good catch!

Reply:
i think the latest commit reverted the fix?